ground-core/scripts/cassandra/cassandra.sql

```diff
@@ -1,134 +1,136 @@
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table StructureVersions (
+create table structure_version (
     id varchar primary key,
     structure_id varchar
 );
 
-create table StructureVersionItems (
-    svid varchar,
+create table structure_version_attribute (
+    structure_version_id varchar,
     key varchar,
     type varchar,
-    primary key(svid, key)
+    PRIMARY KEY (structure_version_id, key)
 );
 
-create table RichVersions (
-    id varchar primary key,
-    structure_id varchar,
+create table rich_version (
+    id varchar PRIMARY KEY,
+    structure_version_id varchar,
     reference varchar
 );
 
-create table RichVersionExternalParameters (
-    richversion_id varchar,
+create table rich_version_external_parameter (
+    rich_version_id varchar,
     key varchar,
     value varchar,
-    primary key (richversion_id, key)
+    PRIMARY KEY (rich_version_id, key)
 );
 
-create table Tags (
-    richversion_id varchar,
+create table tag (
+    rich_version_id varchar,
     key varchar,
     value varchar,
     type varchar,
-    primary key(richversion_id, key)
+    PRIMARY KEY (rich_version_id, key)
 );
 
-create table Edges (
+create table edge (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table Nodes (
+create table node (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table Graphs (
+create table graph (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
 );
 
-create table NodeVersions (
-    id varchar primary key,
-    node_id varchar,
+create table node_version (
+    id varchar PRIMARY KEY,
+    node_id varchar
 );
 
-create table EdgeVersions (
-    id varchar primary key,
+create table edge_version (
+    id varchar PRIMARY KEY,
     edge_id varchar,
-    endpoint_one varchar,
-    endpoint_two varchar,
+    from_node_version_id varchar,
+    to_node_version_id varchar
 );
 
-create table GraphVersions (
+create table graph_version (
     id varchar primary key,
     graph_id varchar,
 );
 
-create table GraphVersionEdges (
-    gvid varchar,
-    evid varchar,
-    primary key(gvid, evid)
+create table graph_version_edge (
+    graph_version_id varchar,
+    edge_version_id varchar,
+    PRIMARY KEY (graph_version_id, edge_version_id)
 );
 
 /* USAGE */
 
-create table Workflows (
+create table workflow (
     graph_id varchar,
     name varchar,
-    primary key(graph_id, name)
+    PRIMARY KEY (graph_id, name)
 );
 
-create table Principals (
+create table principal (
     node_id varchar,
     name varchar,
-    primary key(node_id, name)
+    PRIMARY KEY (node_id, name)
 );
 
-create table LineageEdges (
+create table lineage_edge (
     item_id varchar,
     name varchar,
-    primary key(item_id, name)
+    PRIMARY KEY (item_id, name)
```

Comment:
Same comment as above about the `name` attribute. Does not seem to belong to the primary key (In Cassandra, it becomes the cluster key, effectively creating an internal structure as: `(item_id, name) -> timestamp` that does not seem to be required)